cava/alexa.cava

```diff
@@ -0,0 +1,8 @@
+public static void main(String[] args) {
+
+  char *phrase = "This is so sad, Alexa play dispacito.";
+  printf("%s", phrase);
+  return 0;
+  
```

Comment:
Remove the newlines on lines 2 and 6. Then it LGTM!

Reply:
extra lines removed boss.